Prace_domowe/Praca_domowa2/Grupa4/Yevhenii_Vinichenko/pd2.ipynb

```diff
@@ -0,0 +1,1446 @@
+{
```

Comment:
<p>Oczywiście overfitting nie wynika z tego, że liczymy średnią, ale może wynikać z tego, że korzystamy z targetu, ale zakładam, że to wiesz \;)</p><p> </p><p>Dobrze, że jest lower</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>